enterprise-uda-installer-archives-mapping.ttl

```diff
@@ -8583,9 +8583,9 @@ source: a schema:CreativeWork ;
         <http://data.openlinksw.com/oplweb/product_release/UDAMultiTierEnterpriseEditionJDBCRelease7Oracle#this>,
         <http://data.openlinksw.com/oplweb/product_release/UDAMultiTierEnterpriseEditionODBCRelease7Oracle#this> ;
     oplpro:versionText "7.0" ;
-    oplsof:hasDatabaseEngine oplsof:Oracle12 ;
+    oplsof:hasDatabaseEngine oplsof:Oracle12, oplsof:Oracle19, oplsof:Oracle21 ;
     oplsof:hasDatabaseFamily oplsof:Oracle ;
-    oplsof:hasDbmsEngineVersion "12.x" ;
+    oplsof:hasDbmsEngineVersion "12.x", "19c", "21c" ;
```

Comment:
It seems to me that these should be consistent -- either all [Oracle's branding](https://www.orafaq.com/wiki/Oracle_database), or all *versions* -- i.e., `
```suggestion
    oplsof:hasDbmsEngineVersion "12.x", "19.x", "21.x" ;
```

Reply:
*OR*
```suggestion
    oplsof:hasDbmsEngineVersion "12c", "19c", "21c" ;
```

Reply:
I think whichever of those is chosen here should then be applied throughout.

Reply:
Can't you make the change? The latter suggestion is preferred.

/cc @JacquiHand

Reply:
I believe @JacquiHand should be able to quickly and easily make these changes and update this PR via a search-and-replace on her fork.

I can make change suggestions on this PR, but it will take much longer, as I must make such a suggestion on each occurrence.

Or, I can do a search-and-replace against the main branch after this PR is merged.

Reply:
@TallTed , 

Merged, so you can perform the global replace and then generate another pull request. 

/cc @JacquiHand